docs/manifest.json

```diff
@@ -1,212 +1,254 @@
+    "parent": "reference"
+  },
+  {
+    "title": "Design Principles",
+    "slug": "design-principles",
+    "markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/reference/design-principles.md",
+    "parent": "reference"
+  },
+  {
+    "title": "History",
+    "slug": "history",
+    "markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/reference/history.md",
+    "parent": "reference"
+  },
+  {
+    "title": "Coding Guidelines",
+    "slug": "coding-guidelines",
+    "markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/reference/coding-guidelines.md",
+    "parent": "reference"
+  },
+  {
+    "title": "Testing Overview",
+    "slug": "testing-overview",
+    "markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/reference/testing-overview.md",
+    "parent": "reference"
+  },
+  {
+    "title": "Frequently Asked Questions",
+    "slug": "faq",
+    "markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/reference/faq.md",
+    "parent": "reference"
+  },
+  {
+    "title": "Release Process",
+    "slug": "release",
+    "markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/reference/release.md",
+    "parent": "reference"
+  },
+  {
+    "title": "Deprecated Features",
+    "slug": "deprecated",
+    "markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/reference/deprecated.md",
+    "parent": "reference"
+  },
+  {
+    "title": "Repository Management",
+    "slug": "repository-management",
+    "markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/reference/repository-management.md",
+    "parent": "reference"
+  },
+  {
+    "title": "Outreach",
+    "slug": "outreach",
+    "markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/outreach.md",
+    "parent": null
+  },
+  {
+    "title": "Articles",
+    "slug": "articles",
+    "markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/outreach/docs/articles.md",
+    "parent": "outreach"
+  },
+  {
+    "title": "Meetups",
+    "slug": "meetups",
+    "markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/outreach/meetups.md",
+    "parent": "outreach"
+  },
+  {
+    "title": "Talks",
+    "slug": "talks",
+    "markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/outreach/talks.md",
+    "parent": "outreach"
+  },
+  {
+    "title": "Resources",
+    "slug": "resources",
+    "markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/outreach/resources.md",
+    "parent": "outreach"
+  },
+  {
+    "title": "Data Package Reference",
+    "slug": "packages-data",
+    "markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/data/index.md",
```

Comment:
Can this file be renamed to README.md to enable the default GitHub support in the browser when browsing repository?